packages/node_modules/@webex/internal-plugin-conversation/test/unit/spec/conversation.js

```diff
@@ -707,5 +707,67 @@ describe('plugin-conversation', () => {
         }
       });
     });
+
+    describe('delete()', () => {
+      const testConvo = {
+        id: 'id1',
+        url: 'https://example.com',
+      };
+
+      it('should reject if provided param is not an object', () => {
+        const request = webex.internal.conversation.delete(testConvo, 'hello');
+
+        return request.then(() => {
+          assert.equal(true, false, 'should have rejected');
+        })
+          .catch(() => {
+            assert.equal(true, true, 'object is not type object, rejects as expected');
+          });
+      });
+      it('deletes a non-meeting container activity', () => {
+        webex.internal.conversation.prepare = sinon.stub().callsFake((activity, request) => Promise.resolve({activity, request}));
+        webex.internal.conversation.submit = sinon.stub().callsFake((p) => Promise.resolve(p));
+
+        // fix this to look like below
+        const request = webex.internal.conversation.delete(testConvo, {
+          id: 'activity-id-1',
+          url: 'https://example.com/activity1',
+          object: {objectType: 'activity'}
+        },
+        {
+          object:
+          {objectType: 'activity'}
+        });
+
+        return request.then(({request}) => {
+          assert.isUndefined(request.kmsMessage);
```

Comment:
question: Is it worth checking here that the `request.target.kmsResourceObjectUrl` hasn't been modified?